stages/org.osbuild.anaconda

```diff
@@ -25,6 +25,30 @@ SCHEMA = """
       "type": "string"
     },
     "minItems": 1
+  },
+  "optional-modules": {
+    "type": "array",
+    "description": "Optional modules to enable",
+    "items": {
+      "type": "string"
+    },
+    "minItems": 1
+  },
+  "forbidden-modules": {
+    "type": "array",
+    "description": "Modules to forbid",
+    "items": {
+      "type": "string"
+    },
+    "minItems": 1
+  },
+  "activatable-modules": {
+    "type": "array",
+    "description": "Modules to activate",
+    "items": {
+      "type": "string"
+    },
+    "minItems": 1
```

Comment:
Does `"required": ["kickstart-modules"]` still make sense? That is, are these extra configurations necessarily dependent on configures kickstart modules?

Reply:
We could maybe link to the config file, https://anaconda-installer.readthedocs.io/en/latest/configuration-files.html

To your question:
```
# List of enabled Anaconda DBus modules.
# This option is deprecated and will be removed in in the future.
# kickstart_modules =
```

Reply:
It doesn't make sense anymore, in fact it shouldn't be used at all anymore (but it still can be used).